website/docs/src/main/resources/microsite/data/menu.yml

```diff
@@ -172,6 +172,10 @@ options:
         url: chisel3/docs/explanations/properties.html
         menu_type: chisel3
         menu_section: properties
+      - title: Optional Groups
```

Comment:
thanks for adding this here. Can you add it in the same place on the ToC page of the /docs/src/explanations/explanations.md?